packages/slidev/node/cli.ts

```diff
@@ -611,6 +613,14 @@ function exportOptions<T>(args: Argv<T>) {
       type: 'number',
       describe: 'scale factor for image export',
     })
+    .option('cover', {
+      type: 'boolean',
+      describe: 'prepend cover to handout, needs handout-cover.vue in project',
+    })
```

Comment:
I suppose we don't need this option? We could do it automatically when `handout-cover.vue` presents.